school/lms/doctype/cohort/cohort.json

```diff
@@ -0,0 +1,142 @@
+{
+ "actions": [],
+ "allow_rename": 1,
+ "autoname": "format:{course}/{slug}",
+ "creation": "2021-11-19 11:45:31.016097",
+ "doctype": "DocType",
+ "editable_grid": 1,
+ "engine": "InnoDB",
+ "field_order": [
+  "course",
+  "slug",
+  "title",
+  "section_break_2",
+  "instructor",
+  "status",
+  "column_break_4",
+  "begin_date",
+  "end_date",
+  "duration",
+  "section_break_8",
+  "description",
+  "pages"
+ ],
+ "fields": [
+  {
+   "fieldname": "description",
+   "fieldtype": "Markdown Editor",
+   "label": "Description"
+  },
+  {
+   "fieldname": "instructor",
+   "fieldtype": "Link",
+   "label": "Instructor",
+   "options": "User",
+   "reqd": 1
+  },
+  {
+   "fieldname": "status",
+   "fieldtype": "Select",
+   "label": "Status",
+   "options": "Upcoming\nLive\nCompleted\nCancelled",
+   "reqd": 1
+  },
+  {
+   "fieldname": "begin_date",
+   "fieldtype": "Date",
+   "label": "Begin Date"
+  },
+  {
+   "fieldname": "end_date",
+   "fieldtype": "Date",
+   "label": "End Date"
+  },
+  {
+   "fieldname": "duration",
+   "fieldtype": "Data",
+   "label": "Duration"
+  },
+  {
+   "fieldname": "slug",
+   "fieldtype": "Data",
+   "in_list_view": 1,
+   "label": "Slug",
+   "reqd": 1,
+   "unique": 1
+  },
+  {
+   "fieldname": "column_break_4",
+   "fieldtype": "Column Break"
+  },
+  {
+   "fieldname": "section_break_8",
+   "fieldtype": "Section Break"
+  },
+  {
+   "fieldname": "section_break_2",
+   "fieldtype": "Section Break"
+  },
+  {
+   "fieldname": "title",
+   "fieldtype": "Data",
+   "in_list_view": 1,
+   "label": "title",
+   "reqd": 1
+  },
+  {
+   "fieldname": "course",
+   "fieldtype": "Link",
+   "in_list_view": 1,
+   "label": "Course",
+   "options": "LMS Course",
+   "reqd": 1
+  },
+  {
+   "fieldname": "pages",
+   "fieldtype": "Table",
+   "label": "Pages",
+   "options": "Cohort Web Page"
+  }
+ ],
+ "index_web_pages_for_search": 1,
+ "links": [
+  {
+   "group": "Links",
+   "link_doctype": "Cohort Subgroup",
+   "link_fieldname": "cohort"
+  },
+  {
+   "group": "Links",
+   "link_doctype": "Cohort Student",
```

Comment:
This Doctype is missing. Is this in Mon School? It's causing an error when I tried to make changes in Cohort Subgroup

Reply:
No, this should go away. Not sure how migration was successful on mon school. I'll remove it.

Reply:
Resolved this issue in eb59713b656fc33106dd792406bdb0d42f6575d6.